src/interfaces/lists/IWorkflowNodes.ts

```diff
@@ -0,0 +1,13 @@
+export interface IWorkflowNodes {
+  asyncBefore?: boolean;
+  description?: string;
+  deserializeAs?: string;
+  expression?: string;
+  id?: string;
+  identifier?: string;
+  name?: boolean;
+  type?: string;
+  interrupting?: boolean;
+  _links?: Object;
+  page: Object;
+}
```

Comment:
We may not want a specific type for these.
The structure is highly subject to change.

I recommend not having an explicit type here.
We can still access the object properties without an explicit type.